### PR TITLE
2.13.0-RC1

### PR DESCRIPTION
**Pull request by travisbrown**
This will need updating when Claimant is released, but I confirmed that it works locally with `sbt +test` and `sbt +publishLocal`. I've removed the Rojoma support modules from aggregation (they're the only ones with no 2.13 support), so they'll need to be published manually, which is annoying but seems better than publishing the 2.13.0-RC1 artifacts manually.

### CODE REVIEW

**Comment by travisbrown** [comment]
Okay, this is ready for review! /cc @non @rossabaker?

File: build.sbt [hunk 1]
```diff
@@ -173,4 +165,4 @@ lazy val benchmark = project.in(file("benchmark"))
   .enablePlugins(JmhPlugin)
 
 lazy val all =
-  Seq(parser, util, ast, supportArgonaut, supportJson4s, supportPlay, supportRojoma, supportRojomaV3, supportSpray)
+  Seq(parser, util, ast, supportArgonaut, supportJson4s, supportPlay, supportSpray)
```
Comment (rossabaker) [inline build.sbt]: We made automatic fully automated publishing work on http4s with a missing boopickle module, but it took a couple days and was totally not worth it.

**Comment by travisbrown** [comment]
Just resolved after the other PRs, will merge when green.

**Comment by rossabaker** [comment]
I've never seeen rojoma in the wild.  I'm not passing judgement on its quality (never tried it), but if it's complicating our build, it's worth considering whether it still belongs in this repo.

**Comment by travisbrown** [comment]
There are no changes to the source since 0.14.1 but we did bump patch versions of some of the support deps (necessarily for 2.13.0-RC1) so I think it makes sense to publish everything as 0.14.2 instead of just a 0.14.1 for 2.13.0-RC1.

**Comment by travisbrown** [comment]
@rossabaker Agreed. Want to open an issue to get feedback? In any case we could probably skip 0.14.2 for Rojoma (since those would be identical to 0.14.1) and see if anyone complains.

**Comment by travisbrown** [comment]
Weird:

```scala
[info] ! ParseLongCheck.safe parser fails on invalid input: Falsified after 12 passed tests.
[info] > Labels of failing property: 
[info] falsified: false
[info] > ARG_0: 866772140716548367
[info] > ARG_1: 2370320919339447806
[info] > ARG_2: "༩"
```

I'll take a look in the morning and probably publish then.

**Comment by travisbrown** [comment]
Restarted the build and opened #163 for the test failure.

**Pull request merged**